frontend/packages/console-shared/src/components/catalog/CatalogController.tsx

```diff
@@ -174,7 +190,7 @@ const CatalogController: React.FC<CatalogControllerProps> = ({
       <div className="co-m-page__body">
         <div className="co-catalog">
           <PageHeading title={title} breadcrumbs={type ? breadcrumbs : null} />
-          <p className="co-catalog-page__description">{description}</p>
+          {getCatalogDescription()}
```

Comment:
@debsmita1 Just read the code, and all three cases adds `<p className="co-catalog-page__description">` around the component they render. Two times in `getCatalogDescription` and the last time in `helmCatalogTypeDescription`.

Is there any reason to not keep it here and make the return statements of `getCatalogDescription` and `helmCatalogTypeDescription` easier?

Reply:
Fixed it